api/db.py

```diff
@@ -65,6 +72,20 @@ async def find_by_id(self, model, obj_id):
         obj = await col.find_one({'_id': ObjectId(obj_id)})
         return model(**obj) if obj else None
 
+    def _operator_translation(self, attributes):
+        translated = attributes.copy()
+        for key in attributes.keys():
+            field = key.split('__')
+            if len(field) == 2:
+                if not self.operator_map.get(field[1]):
+                    raise ValueError(f"No operator found matching \
+'{field[1]}'")
```

Comment:
It's much better to avoid wrapping strings, it's harder to read when the line goes back to column 0 and it makes it harder to search for messages in the code.  This should pass `pycodestyle` too:
```python
                    raise ValueError(
                        f"No operator found matching '{field[1]}'"
                    )
```